packages/plugin/src/Bundles/Form/HiddenInputs/CsrfInput.php

```diff
@@ -17,21 +18,13 @@ public function __construct()
             return;
         }
 
-        if (!\Craft::$app->request->isConsoleRequest) {
-            // Prevent response from being cached with token
-            \Craft::$app->getResponse()->setNoCacheHeaders();
-        }
-
```

Comment:
Nocache headers are automatically sent when retrieving/generating a CSRF token in Craft 5.2.6/4.10.5